packages/core/src/routes/experience/verification-routes/enterprise-sso-verification.ts

```diff
@@ -100,4 +100,36 @@ export default function enterpriseSsoVerificationRoutes<T extends WithLogContext
       return next();
     }
   );
+
+  router.get(
+    `${experienceRoutes.verification}/sso/connectors`,
```

Comment:
not sure if this should be put under `/verification` or just `/sso`

Reply:
All SSO endpoints are listed under the verification routes,  a stand-alone `/sso` seems not proper as well.